src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const SUM = days * PRICE_PER_DAY;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return SUM - LONG_TERM_DISCOUNT;
+  }
```

Comment:
The use of 'if' with 'return' here is good, as it simplifies the logic and avoids unnecessary 'else' statements.